<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QListWidget" name="listContact"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>nombre Contact:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="nbC">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="AjouterInteractionButton">
          <property name="text">
           <string>Ajouter Interaction</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="AfficheInteractionButton">
          <property name="text">
           <string>Afficher Interaction</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="AfficheTaches">
          <property name="text">
           <string>Afficher Taches</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="labelDu">
        <property name="text">
         <string>du</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateDu"/>
      </item>
      <item>
       <widget class="QLabel" name="labelAu">
        <property name="text">
         <string>au</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateAu"/>
      </item>
      <item>
       <widget class="QPushButton" name="showButton">
        <property name="text">
         <string>Afficher</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>479</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionAjouter_contact"/>
    <addaction name="actionRecherche"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAjouter_contact">
   <property name="text">
    <string>Ajouter contact</string>
   </property>
  </action>
  <action name="actionRecherche">
   <property name="text">
    <string>Recherche contact</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
